### PR TITLE
Remove most CODEOWNERS

### PR DESCRIPTION
**Pull request by stuartmorgan**
In practice, the review response rate for these auto-assignments is generally quite low. The result is that we frequently have PRs with reviewers assigned but who never actually review; this makes following up on PR status in PR triage more difficult, and can create confusion for new contributors (who believe their PR has a review when, functionally, it doesn't).

## Pre-launch Checklist

- [x] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [x] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [x] I read and followed the [relevant style guides] and ran [the auto-formatter]. (Unlike the flutter/flutter repo, the flutter/plugins repo does use `dart format`.)
- [x] I signed the [CLA].
- [ ] The title of the PR starts with the name of the plugin surrounded by square brackets, e.g. `[shared_preferences]`
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated `pubspec.yaml` with an appropriate new version according to the [pub versioning philosophy], or this PR is [exempt from version changes].
- [ ] I updated `CHANGELOG.md` to add a description of the change, [following repository CHANGELOG style].
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [x] I added new tests to check the change I am making, or this PR is [test-exempt].
- [x] All existing and new tests are passing.

[Contributor Guide]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[relevant style guides]: https://github.com/flutter/plugins/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning
[exempt from version changes]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#version-and-changelog-updates
[following repository CHANGELOG style]: https://github.com/flutter/flutter/wiki/Contributing-to-Plugins-and-Packages#changelog-style
[the auto-formatter]: https://github.com/flutter/plugins/blob/master/script/tool/README.md#format-code
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests

### CODE REVIEW

**Comment by stuartmorgan** [comment]
@bparrishMines @ditman @cyanglaz @LHLL @gaaclarke This is a proposal, we don't necessarily need to do the full removal. However, I would like anyone in this file to commit to **actively monitoring** their auto-assigned review requests. That means that when auto-assigned you should always either:
- actually review the PR, or
- promptly update the PR to remove yourself as a reviewer if you aren't planning to review it

If there are packages for which you want to commit to that, please comment here to let me know what those are. (E.g., @cyanglaz maybe you'd want to narrow your assignments to iOS implementations of some package?)

Not commenting will be assumed to mean you aren't actively monitoring PRs here, in which case I'll remove you from the file 🙂

**Review by ditman — APPROVED** [review]
Not opposed to this removing the file, also not opposed to adding myself as:

```gitignore
**/*_web/**  @ditman
```

(not sure if those glob patterns would work, basically monitor anything that ends up in `_web`)

**Review by cyanglaz — APPROVED** [review]
LGTM

**Review by bparrishMines — APPROVED** [review]
LGTM

I would also be fine with committing to `webview_flutter` PRs since I've been working across the entire plugin for the last couple months.

**Review by gaaclarke — APPROVED** [review]

**Comment by stuartmorgan** [comment]
I've updated this to leave the file in place with just the commitments mentioned above. We'll see how that goes!

File: CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Below is a list of Flutter team members who are suggested reviewers
+# for contributions to plugins in this repository.
+#
+# These names are just suggestions. It is fine to have your changes
+# reviewed by someone else.
+
+packages/**/*_web/**                           @ditman
```
Comment (ditman) [inline CODEOWNERS]: I hope this works as advertised! I'll keep an eye out on my auto-assigned issues!

**Review by ditman — APPROVED** [review]
LGTM!

File: CODEOWNERS (hunk 1 above)
Comment (ditman) [reply CODEOWNERS]: It seems to be working! Very nice!

**Pull request merged**